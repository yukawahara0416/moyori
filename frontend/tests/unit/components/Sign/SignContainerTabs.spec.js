import { shallowMount } from '@vue/test-utils'
import Component from '@/components/Sign/SignContainerTabs.vue'

let wrapper
let propsData

beforeEach(() => {
  propsData = {
    tabs: 'tab-1'
  }

  wrapper = shallowMount(Component, {
    propsData
  })
})

afterEach(() => {
  wrapper.destroy()
})

describe('props', () => {
  it('tabs', () => {
    expect(wrapper.props().tabs).toEqual(propsData.tabs)
  })
})

// describe('computed', () => {
//   it('childTabs', () => {})
// })

describe('template', () => {
  it('snapshot', () => {
    expect(wrapper.vm.$el).toMatchSnapshot()
  })
})
