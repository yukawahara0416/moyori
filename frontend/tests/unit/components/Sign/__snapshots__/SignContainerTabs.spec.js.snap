// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`template snapshot 1`] = `
<v-tabs-stub
  activeclass=""
  fixedtabs="true"
  mobilebreakpoint="1264"
  nexticon="$next"
  previcon="$prev"
  slidersize="2"
  value="tab-1"
>
  <v-tab-stub
    href="#tab-1"
    ripple="true"
  >
    <v-icon-stub
      left=""
    >
      mdi-login
    </v-icon-stub>
    
    ログイン
  
  </v-tab-stub>
   
  <v-tab-stub
    href="#tab-2"
    ripple="true"
  >
    <v-icon-stub
      left=""
    >
      mdi-account-plus
    </v-icon-stub>
    
    新規登録
  
  </v-tab-stub>
</v-tabs-stub>
`;
